applications/StructuralMechanicsApplication/custom_constitutive/linear_j2_plasticity_3d.cpp

```diff
@@ -93,6 +93,21 @@ void LinearJ2Plasticity3D::SetValue(
 //************************************************************************************
 //************************************************************************************
 
+double& LinearJ2Plasticity3D::GetValue(
+    const Variable<double>& rThisVariable,
+    double& rValue
+    )
+{
+    if(rThisVariable == PLASTIC_STRAIN){
+        rValue = mAccumulatedPlasticStrain;
```

Comment:
Shouldn't this throw a warning or something if the variable is incorrect? (Just asking I have no idea)

Reply:
There is another function Has, the user should check first with the Has... (this is part of the design)